woocommerce/woocommerce-ios
# [Local notification] Remove 24h notification remote feature flag

Closes: #10266 

## Description

The deprecated local notification's remote feature flag `woo_notification_24h_after_free_trial_subscribed` is being removed in D117388-code

This PR removes the iOS side code which parses that remote feature flag. 

## Testing instructions
Launch the app and do a smoke test to ensure that the app works as before without this remote feature flag. 

## Screenshots
NA

---
